daemon/images/images_test.go

```diff
@@ -0,0 +1,102 @@
+package images
+
+import (
+	"io/ioutil"
+	"os"
+	"runtime"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/image"
+	"github.com/docker/docker/layer"
+	"github.com/docker/docker/reference"
+	"github.com/gotestyourself/gotestyourself/assert"
+)
+
+var marshalledSaveLoadTestCases = []byte(`{"Repositories":{"busybox":{"busybox:latest":"sha256:1f86fbc76ed941d216a0639c8d787b586c21a2beca6e4dcc28abc9172bc2d6dc"},"jess/hollywood":{"jess/hollywood:latest":"sha256:472de72579154d792ace622ef45572124819d668624ab847b29dcaaf71f79f5d"},"registry":{"registry@sha256:367eb40fd0330a7e464777121e39d2f5b3e8e23a1e159342e53ab05c9e4d94e6":"sha256:24126a56805beb9711be5f4590cc2eb55ab8d4a85ebd618eed72bb19fc50631c"},"registry:5000/foobar":{"registry:5000/foobar:HEAD":"sha256:470022b8af682154f57a2163d030eb369549549cba00edc69e1b99b46bb924d6","registry:5000/foobar:alternate":"sha256:ae300ebc4a4f00693702cfb0a5e0b7bc527b353828dc86ad09fb95c8a681b793","registry:5000/foobar:latest":"ha256:ed5db16db354a7facad7f953d708fd4dc9f9dd3653bb66ec621f4c35d380e64a","registry:5000/foobar:master":"sha256:6c9917af4c4e05001b346421959d7ea81b6dc9d25718466a37a6add865dfd7fc"}}}`)
```

Comment:
`ha256:ed5db16db354a7facad7f953d708fd4dc9f9dd3653bb66ec621f4c35d380e64a` there is a typo there :sweat_smile: